verify/verify.go

```diff
@@ -772,6 +770,23 @@ func GetAttestationFromReport(report *spb.Report, options *Options) (*spb.Attest
 	if err := fillInAttestation(result, options); err != nil {
 		return nil, err
 	}
+	// Attempt to fill in the product field of the attestation. Don't error at this
+	// point since this is not validation.
+	info, _ := abi.ParseSignerInfo(report.SignerInfo)
+	var exts *kds.Extensions
+	parse := func(der []byte) *x509.Certificate {
+		out, _ := x509.ParseCertificate(der)
```

Comment:
nit: It makes sense not to error here. But I wonder if there's an error from ParseCertificate (or kds.ParseProductName), shall we add a log for them? Or are we relying on the real error message to include those info?

Reply:
The real error message will happen during verify.SnpAttestation if a certificate doesn't parse.

Reply:
Is there a change you're requesting?

Reply:
SGTM.